app/presenters/flow_presenter.rb

```diff
@@ -94,6 +94,27 @@ def start_node
     @start_node ||= StartNodePresenter.new(node)
   end
 
+  def previous_state
+    unless current_state.path.empty?
+      presenter_for(@flow.node(current_state.path.last))
+    end
+  end
+
+  def back_link(question_number)
```

Comment:
Ahh I see we introduced this question_number parameter. I don't think we actually need this as a caller parameter since this presenter has a concept of current and back is contextual to that.

I think we could get this from `current_question_number - 1` within this method.

Reply:
thanks! I got rid of the argument.